tools/cassandra/README.md

```diff
@@ -7,18 +7,30 @@ How
 - Run make bins
 - You should see an executable `cadence-cassandra-tool`
 
-Setting up initial cassandra schema on a new cluster
+Setting up cassandra schema on a new cluster using Makefile
+----------------------------------------------------
+``` 
+make install-schema
+```
+
+Setting up schema on a new cluster manually
```

Comment:
I don't think this section is necessary If folks want to dig in, they can easily look up Makefile to see command breakdown.